src/neuroglancer/datasource/dvid/frontend.ts

```diff
@@ -91,34 +111,50 @@ export class VolumeDataInstanceInfo extends DataInstanceInfo {
 
   get volumeType() {
     return (
-        this.encoding === VolumeChunkEncoding.COMPRESSED_SEGMENTATION ? VolumeType.SEGMENTATION :
-                                                                        VolumeType.IMAGE);
+        (this.encoding === VolumeChunkEncoding.COMPRESSED_SEGMENTATION ||
+         this.encoding === VolumeChunkEncoding.COMPRESSED_SEGMENTATIONARRAY) ?
+            VolumeType.SEGMENTATION :
+            VolumeType.IMAGE);
   }
 
   getSources(
       chunkManager: ChunkManager, parameters: DVIDSourceParameters,
       volumeSourceOptions: VolumeSourceOptions) {
     let {encoding} = this;
     let sources: VolumeChunkSource[][] = [];
+
+    // must be 64 block size to work with neuroglancer properly
+    let blocksize = 64;
```

Comment:
Can you explain what this blocksize alignment solves?

Reply:
Before this modification, there were some circumstances where the lower bound was not coming back as multiple of the chunk size (set to 64).  I think this was occurring because in DVID, the dataset doesn't have to start at 0,0,0 (and can even be negative) which seems to introduce a small off-by-one issue.

Reply:
In principle neuroglancer supports arbitrary bounds non-chunk-aligned (or even non-voxel-aligned via the separate "clip" bounds), even negative ones, but perhaps there is some bug somewhere.  In any case I've merged this and we can address this issue later perhaps.

Reply:
Thanks Jeremy!  I will do a little sleuthing around to see if there is a bug somewhere deeper in the code.